build/CodeSign/certificate.ignore

```diff
@@ -15,4 +18,6 @@ bin\*\*.wixlib, Ignore wix artifact
 bin\*\*.wixpdb, Ignore wix artifact
 bin\*\*.msidir\*.cab, Ignore embedded cab files
 
-binaries-windows-release\*, Ignore downloaded binary artifacts
\ No newline at end of file
+binaries-windows-release\*, Ignore downloaded binary artifacts
+
+bin\*\newtonsoft.json.dll, Ignore Newtonsoft.Json
```

Comment:
Can you briefly check why we have Newtonsoft.Json in the output? It's probably a test artifact, but I don't recall us doing JSON parsing in managed code.

Reply:
I believe this came in from a dependency package, let me double check

Reply:
Yep, it's a dependency of Microsoft.TestPlatform.TestHost: https://www.nuget.org/packages/Microsoft.TestPlatform.TestHost/16.10.0-preview-20210219-03